deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```diff
@@ -80,74 +83,19 @@ objects:
     - spiaccesstokens
     - spifilecontentrequests
     verbs:
-    - create
-    - get
-    - list
-    - watch
-    - update
-    - patch
-    - delete
-  - apiGroups:
-    - appstudio.redhat.com
-    resources:
-    - spiaccesstokendataupdates
```

Comment:
@skabashnyuk Can you confirm that we don't need users have access to `spiaccesstokendataupdates` resources? They are not listed in https://github.com/redhat-appstudio/book/blob/main/ADR/0011-roles-and-permissions.md

Reply:
@alexeykazakov done in [dda2100](https://github.com/codeready-toolchain/host-operator/pull/775/commits/dda21007d5e2d5209ddd459960eb945d91fbc4e1)